### PR TITLE
Cockpit 270

### PR DESCRIPTION
**Pull request by garrett**
Preview: https://garrett.github.io/cockpit-project.github.io/blog/cockpit-270.html

### CODE REVIEW

**Comment by garrett** [comment]
TODO: We still need a summary and updated links

**Comment by garrett** [comment]
OK, I've quickly added a summary. Now we just need the updated links.

**Comment by garrett** [comment]
Is the Cockpit Machines release 269? Or is it 270?

**Comment by mvollmer** [comment]
> Is the Cockpit Machines release 269? Or is it 270?

269

File: _posts/2022-05-24-cockpit-270.md [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+
+
+## Try it out
+
+Cockpit 270, cockpit-machines 269, and cockpit-podman 48 are available now:
+
+* [For your Linux system](https://cockpit-project.org/running.html)
+
+* [Cockpit Source Tarball](https://github.com/cockpit-project/cockpit/releases/tag/270)
+* [Cockpit Fedora 36](https://bodhi.fedoraproject.org/updates/?releases=F36&packages=cockpit)
```
Comment (garrett) [inline _posts/2022-05-24-cockpit-270.md:65]:
```suggestion
* [Cockpit Fedora 36](https://bodhi.fedoraproject.org/updates/FEDORA-2022-d1b9fc4e30)
```

File: _posts/2022-05-24-cockpit-270.md [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+* [Cockpit Fedora 35](https://bodhi.fedoraproject.org/updates/?releases=F35&packages=cockpit)
+* [Cockpit Client](https://flathub.org/apps/details/org.cockpit_project.CockpitClient)
+* [cockpit-machines Source Tarball](https://github.com/cockpit-project/cockpit-machines/releases/tag/269)
+* [cockpit-machines Fedora 36](https://bodhi.fedoraproject.org/updates/?releases=F36&packages=cockpit-machines)
+* [cockpit-machines Fedora 35](https://bodhi.fedoraproject.org/updates/?releases=F35&packages=cockpit-machines)
+* [Cockpit Client](https://flathub.org/apps/details/org.cockpit_project.CockpitClient)
```
Comment (garrett) [inline _posts/2022-05-24-cockpit-270.md:71]:
```suggestion
```

(Included already; I guess the script outputs it multiple times?)

File: _posts/2022-05-24-cockpit-270.md [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+* [cockpit-podman Source Tarball](https://github.com/cockpit-project/cockpit-podman/releases/tag/48)
+* [cockpit-podman Fedora 36](https://bodhi.fedoraproject.org/updates/?releases=F36&packages=cockpit-podman)
```
Comment (garrett) [inline _posts/2022-05-24-cockpit-270.md:73]:
```suggestion
* [cockpit-podman Fedora 36](https://bodhi.fedoraproject.org/updates/FEDORA-2022-bd43bce242)
```

File: _posts/2022-05-24-cockpit-270.md [hunk 4]
```diff
@@ -0,0 +1,78 @@
...
+* [cockpit-podman Fedora 35](https://bodhi.fedoraproject.org/updates/?releases=F35&packages=cockpit-podman)
```
Comment (garrett) [inline _posts/2022-05-24-cockpit-270.md:74]:
```suggestion
* [cockpit-podman Fedora 35](https://bodhi.fedoraproject.org/updates/FEDORA-2022-60a9b137db)
```

File: _posts/2022-05-24-cockpit-270.md [hunk 5]
```diff
@@ -0,0 +1,78 @@
...
+## Try it out
+
+Cockpit 270, cockpit-machines 269, and cockpit-podman 48 are available now:
+
+* [For your Linux system](https://cockpit-project.org/running.html)
+
+* [Cockpit Source Tarball](https://github.com/cockpit-project/cockpit/releases/tag/270)
+* [Cockpit Fedora 36](https://bodhi.fedoraproject.org/updates/?releases=F36&packages=cockpit)
+* [Cockpit Fedora 35](https://bodhi.fedoraproject.org/updates/?releases=F35&packages=cockpit)
+* [Cockpit Client](https://flathub.org/apps/details/org.cockpit_project.CockpitClient)
```
Comment (garrett) [inline _posts/2022-05-24-cockpit-270.md:67]:
```suggestion
```

**Review by garrett — COMMENTED** [review]
Changed URLs in suggestions

**Comment by garrett** [comment]
OK, I think it's ready. Please look it over one last time.

**Comment by mvollmer** [comment]
LGTM.  All links lead to the correct releases.

**Review by mvollmer — APPROVED** [review]

**Pull request merged**